app/screens/products/index.js

```diff
@@ -1,13 +1,54 @@
-import React from 'react';
-import { View, Text } from 'react-native';
+import React, {Component} from 'react';
+import styles from './styles';
+import {StyleSheet, ScrollView, View} from 'react-native';
+import {Table, Row, Rows} from 'react-native-table-component';
 
-const Products = () => {
+export default class App extends Component {
```

Comment:
Please change class name to ProductsScreen so that it's clear what the page is about